prog/mutation.go

```diff
@@ -183,6 +190,55 @@ func (ctx *mutator) mutateArg() bool {
 	return true
 }
 
+// Select a random argument weighted by its priority
+func (ma *mutationArgs) pseudoRandomArgs(r *randGen) (arg Arg, ctx ArgCtx) {
+
+	// map used to store the indexes of the arguments with the same priority
```

Comment:
This kind of comments should start with a capital letter and end with a dot. Linter doesn't check for it though. Grep for `//` throughout the project to see different kinds of comments and their styles.